salt/modules/linux_lvm.py

```diff
@@ -216,9 +216,11 @@ def pvcreate(devices, override=True, **kwargs):
     '''
     if not devices:
         return 'Error: at least one device is required'
+    if isinstance(devices, str):
```

Comment:
Hello Mike, 
This seems a good suggestion. I didn't know this but what I read about it, it males the code netter portable to python 3. I will make the change and test it, probably after this weekend. 
Kind regards, Luitzen  vrijdag, 20 mei 2016, 06:04PM +02:00 van Mike Place < notifications@github.com> :

>In  salt/modules/linux_lvm.py :
>> @@ -216,9 +216,11 @@ def pvcreate(devices, override=True, **kwargs):
>      '''
>      if not devices:
>          return 'Error: at least one device is required'
> +    if isinstance(devices, str):
>It will be better here to use six.string_types here instead of str, I think.
>—
>You are receiving this because you authored the thread.
>Reply to this email directly or  view it on GitHub